### DISCUSSION TITLE
I accidently create another account with my student email!

### DISCUSSION THREAD

**Discussion by elmurphy** [community/community#140275] [GitHub Education]
### Hi everyone,

Hey guys, there was dramatic moment happend. I wanted to use github student benefits when coding but there is one more thing. I already has my account. So I decided to create another account to use my student benefits but there was a problem I wasn't figure out before it happend. Really to hard manage more than 1 github account. So I wanted use these benefits on my main account but my student mail account is already using. Delete this account and add email to other account might be solution but I read "90 days" warn when I tryed to it. What should I do?

**Comment by angelogalope** [comment]
Maybe try contacting their customer service and address your concern to them. Just let them know that you’re willing to verify both accounts by providing information or proof that those accounts are yours.

**Comment by ucakbeyza** [comment]
You can contact the GitHub team (email), explain this situation, and ask them to add your student email address to your main account. Or you can create an SSH key.
